internal/store/atlas/projects.go

```diff
@@ -43,77 +44,79 @@ type ProjectDescriber interface {
 }
 
 type ProjectUsersLister interface {
-	ProjectUsers(string, *atlas.ListOptions) (interface{}, error)
+	ProjectUsers(string, *atlas.ListOptions) (*atlasv2.PaginatedApiAppUser, error)
 }
 
 type ProjectUserDeleter interface {
 	DeleteUserFromProject(string, string) error
 }
 
 type ProjectTeamLister interface {
-	ProjectTeams(string) (interface{}, error)
+	ProjectTeams(string) (*atlasv2.PaginatedTeamRole, error)
 }
 
 type ProjectTeamAdder interface {
-	AddTeamsToProject(string, []*atlas.ProjectTeam) (*atlas.TeamsAssigned, error)
+	AddTeamsToProject(string, []atlasv2.TeamRole) (*atlasv2.PaginatedTeamRole, error)
 }
 
 type ProjectTeamDeleter interface {
 	DeleteTeamFromProject(string, string) error
 }
 
 // Projects encapsulates the logic to manage different cloud providers.
-func (s *Store) Projects(opts *atlas.ListOptions) (interface{}, error) {
-	result, _, err := s.client.Projects.GetAllProjects(s.ctx, opts)
+func (s *Store) Projects(opts *atlas.ListOptions) (*atlasv2.PaginatedAtlasGroup, error) {
+	result, _, err := s.clientv2.ProjectsApi.ListProjects(s.ctx).PageNum(int32(opts.PageNum)).
+		ItemsPerPage(int32(opts.ItemsPerPage)).IncludeCount(opts.IncludeCount).Execute()
```

Comment:
we don't expose `IncludeCount` in the CLI and assume it should always be included, be careful sice IncludeCount being boolean will default to false in go so to be safe either always pass true here or don't call `IncludeCount` in the store as it should default to true

Reply:
Thanks for the heads-up! 
I believe we have this in a number of places from earlier PRs so going to merge this one and remove all instances in one PR as a follow-up!